### ISSUE TITLE
[Maker community] I want to be able to sort my Spaces

### ISSUE THREAD

**Issue by jonathangarelick** [neevaco/neeva-ios#3390]
From the Maker community:

<img width="382" alt="Screen Shot 2022-04-18 at 9 22 18 AM" src="https://user-images.githubusercontent.com/41495718/163839142-b32b2471-a397-4e22-ac3c-b3c404d70d11.png">

**Comment by jonathangarelick** [comment]
@yozuysal This functionality doesn't exist on the back end, correct?

**Comment by yozuysal** [comment]
No, it doesn't. We allow reordering of Space items, but not Spaces themselves through the API. So potentially this would require an API change (or keeping a local reordered list, which wont carry over to extension uses) and then we will need to figure out UI affordance of how to make this happen. (Drag and drop with LazyGrid or a simpler affordance)

**Comment by darinwf** [comment]
There's a design challenge here though. Spaces are ordered by last modification date. If a user explicitly reorders their spaces, then does that mean it disables sorting by last modification date? Or maybe we should offer the user options on how they want to view their spaces?

**Comment by darinwf** [comment]
For tabs, we sort by creation order, which means it is possible to layer on top of that user customized reordering.

**Comment by yozuysal** [comment]
For the latter solution, we do already have a filter button in the bottom toolbar (so far we had intentionally kept it somewhat subtle). Maybe we can explore different ways of filtering through that button and have the choice sticky?

Right now we only suggest, "All spaces" and "Spaces owned by me"
